modules/services/gpg-agent.nix

```diff
@@ -41,7 +41,7 @@ in
 
       home.sessionVariables =
         optionalAttrs cfg.enableSshSupport {
-          SSH_AUTH_SOCK = "\${XDG_RUNTIME_DIR}/gnupg/S.gpg-agent.ssh";
+          SSH_AUTH_SOCK = "\$XDG_RUNTIME_DIR/gnupg/S.gpg-agent.ssh";
```

Comment:
I'm considering adding another fish-specific file to the output instead that replaces `${XDG_RUNTIME_DIR}` with `{$XDG_RUNTIME_DIR}` since `${XDG_RUNTIME_DIR}` is actually valid csh syntax but not valid fish-syntax.